supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -143,6 +144,27 @@ $ oc describe node __<node_name>__
 ----
 ====
 
+.Example AsciiDoc: User-replaced value in an output example
+
```

Comment:
I think you should add the `[subs="+quotes"]` line above here, so people know it's required.

Also then if you indent all of the lines associated with this code block, it will show them all as is. See lines 133-136 for how the above example does it

Reply:
OK, the example starting at ~line 147 now matches the ~line 131 example.

Reply:
LGTM, thanks for the update!